connector/doc/guides/bootstrap_connector.rst

```diff
@@ -208,6 +211,23 @@ created. I propose to create a helper too in
         return checkpoint.add_checkpoint(session, model_name, record_id,
                                          'coffee.backend', backend_id)
 
+Message checkpoint
+------------------
+
+When you need to show a plain text message to a user
+(for instance: an import job did not succeeded),
```

Comment:
I'm not sure than "an import job did not succeeded" is a good example: if a job really failed, its state will be failed and we can't miss it, we don't need to create a checkpoint. 

I was looking for a better example but couldn't find.

For instance: "A product has been imported but could not be attached to the proper category", but in that case you would link the checkpoint to the product AND show the message, don't you think?